components/StateMultipleChoiceQuizPage.tsx

```diff
@@ -1,10 +1,57 @@
-import React from "react";
-import { Text, View } from "react-native";
+import { useTheme } from "@react-navigation/native";
+import React, { useState } from "react";
+import { Text, View, Button } from "react-native";
 
-export default function StateMutipleChoiceQuizPage() {
+type Question = {
+  questionText: string;
+  option: string[];
+  correctAnswer: string;
+};
+const quizQuestions: Question[] = [
+  {
+    questionText: "What is Karnataka's capital?",
+    option: ["Bangalore", "Chennai", "Hyderabad", "Raipur"],
+    correctAnswer: "Bangalore",
+  },
+];
+
+const StateMultipleChoiceQuizApp: React.FC = () => {
```

Comment:
In other files, the syntax we used is different. Let's be consistent.
```suggestion
export default function StateMutipleChoiceQuizPage() {
```

Reply:
I have removed the export statement at the bottom to the top, when the function is being defined, kindly check once